### PR TITLE
[shopsys] nginx binaries are upgraded from 1.14.0 to 1.14.2 in microservices

### PR DESCRIPTION
**Pull request by boris-brtan**
- upgrade is needed to prevent docker build from failing

| Q             | A
| ------------- | ---
|Description, reason for the PR| because of failing build of microservices in monorepo we needed to upgrade version of nginx binaries
|New feature| No 
|BC breaks| No 
|Fixes issues| temporarily since search microservices will be integrated into php-fpm container
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Comment by boris-brtan** [comment]
```bash
docker-compose build --force-rm --no-cache --pull microservice-product-search microservice-product-search-export && docker-compose up -d --force-recreate --build microservice-product-search microservice-product-search-export
```

File: microservices/product-search-export/docker/Dockerfile [hunk 1]
```diff
@@ -2,141 +2,141 @@ FROM php:7.3-fpm-alpine as base
...
-    && mv /tmp/envsubst /usr/local/bin/ \
-    \
-    # Bring in tzdata so users could set the timezones through the environment
-    # variables
-    && apk add --no-cache tzdata \
-    \
-    # forward request and error logs to docker log collector
-    && ln -sf /dev/stdout /var/log/nginx/access.log \
-    && ln -sf /dev/stderr /var/log/nginx/error.log
+	&& CONFIG="\
```
Comment (MattCzerner) [inline microservices/product-search-export/docker/Dockerfile]: can you do it without reindenting the whole file? :)

File: microservices/product-search/docker/Dockerfile [hunk 2]
```diff
@@ -2,141 +2,141 @@ FROM php:7.3-fpm-alpine as base
...
-    && mv /tmp/envsubst /usr/local/bin/ \
-    \
-    # Bring in tzdata so users could set the timezones through the environment
-    # variables
-    && apk add --no-cache tzdata \
-    \
-    # forward request and error logs to docker log collector
-    && ln -sf /dev/stdout /var/log/nginx/access.log \
-    && ln -sf /dev/stderr /var/log/nginx/error.log
+	&& CONFIG="\
```
Comment (MattCzerner) [inline microservices/product-search/docker/Dockerfile]: can you do it without reindenting the whole file? :)

Comment (boris-brtan) [reply microservices/product-search/docker/Dockerfile]: some problem during tabs->spaces conversion

File: microservices/product-search-export/docker/Dockerfile (hunk 1 above)
Comment (boris-brtan) [reply microservices/product-search-export/docker/Dockerfile]: some problem during tabs->spaces conversion

**Comment by PetrHeinz** [comment]
@MattCzerner @boris-brtan I think that `[shopsys]` should be used every time more than one package is changed. I took the liberty to rename the PR, hope that's all-right.

**Comment by boris-brtan** [comment]
i think that this will not be part of Changelog or Upgrade log so there will be no mistake during release and use of changelog linker.

**Comment by boris-brtan** [comment]
microservices integration into php-fpm container were meged

**Pull request closed without merging**